source/unified-test-format/schema-1.4.json

```diff
@@ -55,6 +55,10 @@
             "enum": ["single", "replicaset", "sharded", "sharded-replicaset", "load-balanced"]
           }
         },
+        "serverless": {
+          "type": "string",
+          "enum": ["require", "forbid", "allow"]
```

Comment:
One last thought before we codify this: would "deny" be preferable since that's the inverse of allow in other contexts (e.g. allowlist/denylist)? I don't feel strongly, but curious if others do.

Reply:
I don't really have any strong feelings either on this and would be happy to change it to whatever we think is best.

Reply:
Fine to leave as-is.